src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -1589,18 +1589,21 @@ private CallbackResponse submitSDO(CallbackParams callbackParams) {
         dataBuilder.hearingNotes(getHearingNotes(caseData));
 
         // LiP check ensures any LiP cases will always create takeCaseOffline WA task until CP goes live
```

Comment:
Minor: can this comment be updated to reflect changes for LiP cases when HMC NRO and HMC LiP scenarios are live?

Reply:
Updated comment to reflect a change:
![image](https://github.com/user-attachments/assets/57223dfa-c434-4ee6-8f92-bc55d8dcf9e6)